### ISSUE TITLE
CharacterListItem missing getter for CharacterMeta

### ISSUE THREAD

**Issue by ToshY** [jikan-me/jikan#375]
**Summary**
In v3 it's no longer possible to get important character data like the `malId`, `url`, `images`, and `name` from `CharacterListItem`, as the functionality seems to be replaced with `CharacterMeta`. But there's no getter available for this data, so it's impossible to get character info.

**Solution**
Add getter for character.

```php
/**
 * @return CharacterMeta
 */
public function getCharacter(): CharacterMeta
{
    return $this->character;
}
```

**Version**
`v3.0.0-beta.8`

**Comment by ToshY** [comment]
I took the liberty to add a PR because I actually want this fixed as soon as possible. Is it possible to merge it and tag a new beta release?

**Comment by irfan-dahir** [comment]
Thanks for the PR and good catch! I've merged and released [v3.0.0-beta.9](https://github.com/jikan-me/jikan/releases/tag/v3.0.0-beta.9)
